src/bijectors/planar_layer.jl

```diff
@@ -25,10 +25,8 @@ end
 
 function get_u_hat(u, w)
     # To preserve invertibility
-    return (
-        u + (planar_flow_m(w' * u) - w' * u)[1]
-        * w / (norm(w[:,1],2) ^ 2)
-    )   # from A.1
+    # From A.1
+    return u .+ (planar_flow_m(w' * u) .- w' * u) .* w ./ sum(abs2, w)
```

Comment:
Is it a scalar or 1x1 matrix?

Reply:
If scalar then there is no need for `.-`, just `-` will do.

Reply:
Also not related to this PR but I recommend to keep the broadcasting at the highest level. For example, `planar_flow_m` doesn't need to have broadcasting inside of it. We can broadcast it at call site using `planar_flow_m.(...)`. That way if we have other broadcasting there going on, it will be fused. Same for `dtanh`. In `ψ`, one can then broadcast `dtanh` and fuse all the operations there.

Reply:
I didn't really spend much time thinking about how I could make this better; I just tried making it work for GPU. So thank you for pointing this out!:) I'll implement your suggestions :+1:

> Is it a scalar or 1x1 matrix?

It's a 1x1 matrix.

Reply:
> We are computing w' * u twice.

So these are just 1x1 matrices, i.e. we're just doing scalar multiplication. Is it even worth saving compared to fusing the entire operation together?

Reply:
Why are we using 1x1 matrices instead of scalars?

Reply:
I was wondering the same thing! @sharanry @xukai92 maybe you can answer?
I'm going to put money on the reason being to be compatible with `Tracker` in the sense that you can do `RadialLayer(n, Tracker.param)`, then take gradients wrt. parameters of `RadialLayer` and update inplace. Can't do inplace updates if they're `<:Real` (without making the entire struct mutable).